src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -984,6 +1017,51 @@ private void ThrowIfWebhooksNotConfigured()
             }
         }
 
+        internal bool TryGetRpcBaseUrl(out Uri rpcBaseUrl)
+        {
+            if (this.config.Options.LocalRpcEndpointEnabled != false)
+            {
+                rpcBaseUrl = InternalRpcUri;
+                return true;
+            }
+
+            // The app owner explicitly disabled the local RPC endpoint.
+            rpcBaseUrl = null;
+            return false;
+        }
+
+#if !FUNCTIONS_V1
+        internal async Task StartLocalHttpServerAsync()
+        {
+            if (!this.localHttpListener.IsListening)

```

Comment:
Why can't we do these checks and logging inside of `LocalHttpListener.Start/StopAsync()` and make those methods idempotent. Then we don't need to do the conditional compilation guards around this method or in DurableTaskExtension.

Reply:
That's possibly true, but I'm afraid of accidentally breaking v1 with this code path, even though I've gone through all the trouble of no-op'ing it. I think at this point I'd prefer to avoid additional churn and will just leave this as-is for now.